src/function/itemLikes.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-return-await */
+/* eslint-disable no-console */
```

Comment:
- [ ] Kindly don't disable linters. Instead, you should manually fix the errors. 😄 🙏 
>Please apply this request to all similar situations through your code files.

Reply:
Issue resolved .. thanks for your feedback 😄 🙏